appveyor.yml

```diff
@@ -6,6 +6,10 @@ stack: go 1.14
 before_test:
   - choco install mingw
   - choco install make
+  - choco install wixtoolset
+  - refreshenv
   - make cross
```

Comment:
`refreshenv`/`make cross` are moved around in the next commit, I guess this could be all done in this commit?

Reply:
Done!